opm/autodiff/MissingFeatures.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+  Copyright 2016 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include <opm/common/OpmLog/OpmLog.hpp>
+#include <opm/parser/eclipse/Deck/DeckKeyword.hpp>
+#include <opm/parser/eclipse/Parser/Parser.hpp>
+#include <opm/parser/eclipse/EclipseState/checkDeck.hpp>
+#include <opm/autodiff/MissingFeatures.hpp>
+#include <vector>
+#include <string>
+
+
+namespace Opm {
+
+    
+    void checkKeywords(DeckConstPtr deck, ParserConstPtr parser)
+    {
+        // These keywords are supported by opm-parser, but are not supported
+        // by flow. For some of them, only part of the options are supported.
+        // The list is used to output messages only.
+        std::vector<std::string> unsupported_keywords = {
```

Comment:
I guess `std::unordered_set` would be a more natural container than `std::vector`